### PR TITLE
(DOCSP-14806): Rename iOS SDK -> Swift SDK

### PR DESCRIPTION
**Pull request by cbush**
### Jira

- https://jira.mongodb.org/browse/DOCSP-14806

### Staged Changes (Requires MongoDB Corp SSO)

- https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/ios-to-swift/sdk/swift/

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/studio/modify-schema.txt [hunk 1]
```diff
@@ -127,7 +127,7 @@ migration guide.
    .. tab::
       :tabid: ios
```
Comment (dacharyc) [inline source/studio/modify-schema.txt:128]: So this is part of tabs-realm-sdks per ln 120 above. This renders as "iOS SDK" on [the page](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/ios-to-swift/studio/modify-schema/). Where do we update that?

**Review by dacharyc — APPROVED** [review]
Overall, this looks great! Just a couple of questions.

Thank you for making this PR!

File: source/stitch.txt [hunk 2]
```diff
@@ -12,7 +12,7 @@ MongoDB Stitch (Legacy)
    (Legacy) Stitch Documentation <http://stitch-docs-old.s3-website-us-east-1.amazonaws.com/>
    (Legacy) Stitch JavaScript Browser SDK <http://stitch-sdks.s3-website-us-east-1.amazonaws.com/stitch-sdks/js/4/index.html>
    (Legacy) Stitch Java/Android SDK <http://stitch-sdks.s3-website-us-east-1.amazonaws.com/stitch-sdks/java/4/overview-summary.html>
-   (Legacy) Stitch Swift/iOS SDK <http://stitch-sdks.s3-website-us-east-1.amazonaws.com/stitch-sdks/swift/6/>
+   (Legacy) Stitch Swift/Swift SDK <http://stitch-sdks.s3-website-us-east-1.amazonaws.com/stitch-sdks/swift/6/>
```
[the remark this answers is not in the document]
Comment (jsflax) [reply source/stitch.txt:15]: We called it the iOS SDK.

**Review by jsflax — APPROVED** [review]

File: config/redirects [hunk 3]
```diff
@@ -283,3 +283,62 @@ raw: realm/sync/protocol -> ${base}/sync/learn/protocol/
...
+raw: realm/sdk/ios/advanced-guides/client-reset -> ${base}/realm/sdk/swift/advanced-guides/client-reset/
+raw: realm/sdk/ios/advanced-guides/compacting -> ${base}/realm/sdk/swift/advanced-guides/compacting/
+raw: realm/sdk/ios/quick-start-with-sync -> ${base}/realm/sdk/swift/quick-start-with-sync/
+raw: realm/sdk/ios/install -> ${base}/realm/sdk/swift/install/
+raw: realm/sdk/ios/test-and-debug -> ${base}/realm/sdk/swift/test-and-debug/
+raw: realm/sdk/ios/advanced-guides -> ${base}/realm/sdk/swift/advanced-guides/
+raw: realm/sdk/ios/additional-examples -> ${base}/realm/sdk/swift/additional-examples/
+raw: realm/sdk/ios/integrations -> ${base}/realm/sdk/swift/integrations/
+raw: realm/sdk/ios/swiftui -> ${base}/realm/sdk/swift/swiftui/
+raw: realm/sdk/ios/integrations -> ${base}/realm/sdk/swift/integrations/
```
Comment (dacharyc) [inline config/redirects:319]: Looks like this one is in here twice - here and ln 317 above.

**Review by dacharyc — APPROVED** [review]
Overall, looks great! I did spot what looks to me like a duplicate redirect, and I also still have that outstanding question about how we rename iOS to Swift in the tabs-realm-sdks - is that a snooty thing that needs to happen outside of our repo? Or is it defined somewhere in the repo?

**Review by dacharyc — CHANGES_REQUESTED** [review]
Oh, looks like I missed a bunch of Snooty build errors - the title underlines are now too short with the longer "Swift" vs. "iOS".

**Review by dacharyc — APPROVED** [review]

**Pull request merged**